tests/FixerConfiguration/AliasedFixerOptionTest.php

```diff
@@ -25,31 +25,104 @@
  */
 final class AliasedFixerOptionTest extends TestCase
 {
-    public function testGetName()
+    /**
+     * @param string             $expected
+     * @param AliasedFixerOption $input
+     *
+     * @dataProvider provideGetNameCases
+     */
+    public function testGetName($expected, AliasedFixerOption $input)
     {
-        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
-        $this->assertSame('foo', $option->getName());
+        $this->assertSame($expected, $input->getName());
     }
 
-    public function testGetDescription()
+    public function provideGetNameCases()
     {
-        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
-        $this->assertSame('Bar.', $option->getDescription());
+        return array(
```

Comment:
You can simplify data set to the bare minimum: the expected name. Since you expect the result to be exactly what you pass to the `FixerOption` constructor, you can use `$expected` as both the assertion's expected value and as `FixerOption` parameter in the test itself. You can probably simplify other data sets the same way.